core/team-members.rst

```diff
@@ -0,0 +1,9 @@
+Team
+----
+
+`Adhityaa Chandrasekar <https://github.com/adtac>`_ (Glorious Leader)
+
+Hi, I'm Adhityaa. I did a GSoC under coala once and I've been here ever
+since. I'm quite familiar with coala's core library (well, duh, I'm the
+leader). I also wrote most of `coala-quickstart` (a tool that is better than
+another tool called `cib`). Also, Arsenal is the greatest club in the world.
```

Comment:
Ok, let's all write Lasse's bio one phrase at a time :P

CC @Adrianzatreanu @Udayan12167 

I'll start: `Hi, I'm Lasse and I like everything open source... `